aspnet-core/xml/FrameworksIndex/aspnetcore-7.0.xml

```diff
@@ -4,8 +4,8 @@
     <Assembly Name="Microsoft.AspNetCore.Antiforgery" Version="7.0.0.0" />
     <Assembly Name="Microsoft.AspNetCore.ApiAuthorization.IdentityServer" Version="7.0.0.0" />
     <Assembly Name="Microsoft.AspNetCore.Authentication.Abstractions" Version="7.0.0.0" />
-    <Assembly Name="Microsoft.AspNetCore.Authentication.AzureAD.UI" Version="7.0.0.0" />
-    <Assembly Name="Microsoft.AspNetCore.Authentication.AzureADB2C.UI" Version="7.0.0.0" />
+    <Assembly Name="Microsoft.AspNetCore.Authentication.AzureAD.UI" Version="6.0.11.0" />
+    <Assembly Name="Microsoft.AspNetCore.Authentication.AzureADB2C.UI" Version="6.0.11.0" />
```

Comment:
Listing 6.0.11 assemblies as if they were part of the 7.0.0 release seems wrong. We don't ship these assemblies anymore.

@HaoK any reason you can think of to continue documenting the APIs from these assemblies❔

@mkArtakMSFT have you and your deprecation tools noted the containing packages aren't part of 7.0❔ I'm thinking deprecating all of the 7.0 preview packages for these two make sense.